.pipelines/pipeline.yaml

```diff
@@ -496,6 +509,12 @@ stages:
             dualstackoverlay_e2e:
               name: dualstackoverlay_e2e
               clusterName: "dsovere2e"
+            cilium_dualstackoverlay_e2e:
+              name: cilium_dualstackoverlay_e2e
+              clusterName: cildsovere2e
+            cilium_dualstackoverlay_cni_e2e:
+              name: cilium_dualstackoverlay_cni_e2e
+              clustername: cildsovcnie2e
```

Comment:
Careful with this cluster name length. Keeping it at 13 characters or under will ensure that the AKS managed cluster name is under 80 characters.